lib/processors/lib/jaccard.py

```diff
@@ -0,0 +1,133 @@
+import random
+
+def jaccard_rec(current_answers):
```

Comment:
@slsp8752 i particularly don't understand the last thing you do where you check question lists for empty responses.... just before the loop you set the questions list from one of those pre-made ones so you know there aren't any missing fields there... or am i missing something?